### PR TITLE
Collaborator add/remove UI

### PR DESCRIPTION
**Pull request by marcelkornblum**

### CODE REVIEW

File: core/forms/irap_request.py [hunk 1]
```diff
@@ -24,6 +25,24 @@ class Meta:
...
+            "DELETE",
+        ]
+        widgets = {
+            "request": forms.HiddenInput(),
+            "DELETE": forms.HiddenInput(),
+        }
+
+    DELETE = forms.BooleanField(
+        required=False,
+    )
```
Comment (SamDudley) [inline core/forms/irap_request.py:43]: not too keen on the caps, isn't obvious what it signifies

File: core/models.py [hunk 2]
```diff
@@ -649,6 +648,10 @@ def is_in_progress(self) -> bool:
     def is_reaccreditation(self) -> bool:
         return self.case_type == self.CaseType.REACCREDITATION
 
+    # @property
+    # def has_collaborator(self) -> bool:
+    #     return len(self.collaborators.all()) > 0
+
```
Comment (SamDudley) [inline core/models.py:654]: needed?

File: core/templates/core/irap_request/task_list.html [hunk 3]
```diff
@@ -40,55 +37,127 @@
...
+        <p>Add team members to help fill parts of the form. They will be notified by email.</p>
 
-<h2>{% lorem 8 w random %}</h3>
-<ul class="govuk-list govuk-list--bullet">
-    <li>{% lorem 8 w random %}</li>
-    <li>{% lorem 8 w random %}</li>
-    <li>{% lorem 8 w random %}</li>
-</ul>
+        <form
+            action="#"
```
Comment (SamDudley) [inline core/templates/core/irap_request/task_list.html:58]: needed?

File: core/templates/core/irap_request/task_list.html [hunk 4]
```diff
@@ -40,55 +37,127 @@
...
+            method="post"
+            novalidate class="user-manage-list"
+            hx-on="htmx:afterSwap: console.log(); this.submit();">
```
Comment (SamDudley) [inline core/templates/core/irap_request/task_list.html:61]: remove console.log

File: core/templates/core/irap_request/task_list.html [hunk 5]
```diff
@@ -40,55 +37,127 @@
...
+                <ul class="task-list">
+            {% endif %}
+            <li>
+                <form
+                    action="#"
+                    method="post"
+                    novalidate>
+                    <input type="hidden" id="user-{{ forloop.counter }}" name="user" value="{{ collaborator.pk }}" />
+                    <input type="hidden" id="request-{{ forloop.counter }}" name="request" value="{{ object.pk }}" />
+                    <input type="hidden" id="DELETE-{{ forloop.counter }}" name="DELETE" value="True" />
```
Comment (SamDudley) [inline core/templates/core/irap_request/task_list.html:83]: why not just the pk of the collaboration model? the through model in between?

File: core/models.py (hunk 2 above)
Comment (marcelkornblum) [reply core/models.py:654]: nope :)

**Review by SamDudley — APPROVED** [review]

**Pull request merged**